src/pages/Profile.js

```diff
@@ -93,6 +93,10 @@ export default function Profile(props) {
             <Submit id="submit" value="Save Information" />
           </Row>
         </Form>
+        <a href="/app">app page</a>
```

Comment:
The AnchorButton at the top of this component already serves this purpose; both don't need to be here.

Reply:
Just realized that in my other open PR handles button logic I am going to remove the hyperlinks from the files I have changed and just keep this commit focused on landing page content only.

Reply:
I will leave the landing page hyperlinks unchanged and opt for buttons in the other files for the sake of continuity.